MD/md-3/README.md

```diff
@@ -1,70 +1,69 @@
-# MD-3: MCR under Network Partitions and Asynchrony
+# MD-3: MCR : Network Partitions and Asynchrony
```

Comment:
this document should be rewritten: 

a) focus on delays rather than network partitions
b) penalization of absenteism makes no sense, so it should be added as a desiderata to assess if it makes sense (unlikely)
c) penalization for absenteism should be removed as a base assumption for the protocol
d) the protocol should be rewritten for postconfirmation (MIP-37)

Reply:
largely done.